FirstProject/evaluation.html

```diff
@@ -16,16 +16,22 @@
     <body>
         <div>
             <div class="row">
+                <!--Navigation bar section-->
                 <nav class="col-md-12" style="border-bottom: 2px solid darkgrey; padding-top: 12px; padding-bottom: 14px;">
```

Comment:
I moved a lot of the styles in CSS files. I made a pull request of the Day03.

Also I moved the styles in .less files.